src/main/java/com/khadri/jdbc/prepare/data/employee/dao/EmployeeInsertDao.java

```diff
@@ -2,19 +2,21 @@
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
+import java.sql.ResultSet;
 import java.sql.SQLException;
+import java.sql.Statement;
 
 import com.khadri.jdbc.prepare.connection.JdbcConnectionUtil;
 import com.khadri.jdbc.prepare.data.employee.dao.model.Employee;
 
 public class EmployeeInsertDao {
```

Comment:
change class name from EmployeeInsertDao to EmployeeDao
add the select and insert methods into it.

Reply:
Resolved